webapp/src/components/home/Home.vue

```diff
@@ -1,45 +1,251 @@
+            Supports arbitrary deep learning architectures
+          </p>
+
+          <span class="text-lg"
+            >Disco supports arbitrary deep learning tasks and model
+            architectures, running on your device via TensorFlow.js</span
+          >
+        </div>
+
+        <div class="lg:w-2/5">
+          <p class="my-3 font-bold text-xlg">Data and model privacy</p>
+          <span class="text-lg">
+            Data privacy by design - no data ever leaves any device. Models
+            updates can be protected by encryption (secure multiparty
+            computation), and by differential privacy.
+          </span>
+        </div>
+
+        <div class="lg:w-2/5">
+          <p class="my-3 font-bold text-xlg">Runs anywhere</p>
+          <span class="text-lg">
+            Disco runs in the browser, from any device equipped with a modern
+            browser. For developers, we also offer a standalone node.js version,
+            not needing any browser.
+          </span>
+        </div>
+      </div>
+    </div>
+    <div class="flex flex-col justify-center">
+      <!-- <Landing v-show="!started" @click="started = true" /> -->
+
+      <ButtonsCard
+        class="lg:w-2/6 mx-auto"
+        title-align="center"
+        :buttons="List.of(['Get started', goToTaskList])"
+      >
+        <template #title>
+          Train a model and contribute to existing
+          <span title="Click Participate to learn more on DISCOllaboratives">
+            <DISCOllaboratives />
+          </span>
+        </template>
+      </ButtonsCard>
     </div>
   </div>
 </template>
-<script lang="ts" setup>
-import { ref } from "vue";
 
-import DiscoGIF from "@/assets/gif/DiscoGIF.vue";
-import Landing from "@/components/home/Landing.vue";
-import GetStarted from "@/components/home/GetStarted.vue";
+<script lang="ts" setup>
 import FeedbackIcon from "@/assets/svg/FeedbackIcon.vue";
 import InfoIcon from "@/assets/svg/InfoIcon.vue";
+import FederatedGIF from "@/assets/gif/FederatedGIF.vue";
+import DecentralizedGIF from "@/assets/gif/DecentralizedGIF.vue";
+import DiscoLogoBoule from "@/assets/svg/DiscoLogoBoule.vue";
+import RealTimeCollaboration from "@/assets/svg/RealTimeCollaboration.vue";
+import { useRouter } from "vue-router";
+import { List } from "immutable";
+import DISCOllaboratives from "@/components/simple/DISCOllaboratives.vue";
+import ButtonsCard from "@/components/containers/ButtonsCard.vue";
+import { driver, type DriveStep } from "driver.js";
+import "driver.js/dist/driver.css";
+
+const router = useRouter();
+
+function goToTaskList(): void {
+  router.push({ path: "/list" });
+}
+// Create a driver object
+const driverObj = driver({
+  showProgress: false,
+});
+
+const steps: DriveStep[] = [
+  {
+    element: "#task-icon",
+    popover: {
+      title: "Collaborate on ML Tasks",
+      description:
+        "DISCOllaboratives let users collaboratively train ML models using shared or personal datasets.",
```

Comment:
we don't (yet?) provide a way to share dataset

```suggestion
        "DISCOllaboratives let users collaboratively train ML models while keeping their datasets private.",
```